extensions/positron-r/src/provider.ts

```diff
@@ -526,6 +532,26 @@ function discoverAdHocBinaries(paths: string[]): RBinary[] {
 		.map(b => ({ path: b, reasons: [ReasonDiscovered.adHoc] }));
 }
 
+/**
+ * Discovers R binaries that are relevant to Posit Workbench.
+ * Paths are from: https://docs.posit.co/ide/server-pro/r/using_multiple_versions_of_r.html
+ * @returns R binaries that are relevant to Posit Workbench.
+ */
+function discoverPWBBinaries(): RBinary[] {
+	const pwbBinaries = discoverHQBinaries([
+		'/usr/lib/R',
+		'/usr/lib64/R',
+		'/usr/local/lib/R',
+		'/usr/local/lib64/R',
+		'/opt/local/lib/R',
+		'/opt/local/lib64/R',
+		// '/opt/R', // Already checked for in rHeadquarters
+		'/opt/local/R'
+	]);
+	// Return the binaries, overwriting the ReasonDiscovered with ReasonDiscovered.pwb
+	return pwbBinaries.map(b => ({ path: b.path, reasons: [ReasonDiscovered.pwb] }));
```

Comment:
Seeing this makes me think we should be doing similar for binaries discovered through user customization! I now suspect we are never actually applying `ReasonDiscovered.user` 🤔

Reply:
we are now! https://github.com/posit-dev/positron/pull/6272/commits/743f89f1157ee31719dd65bfcdee6c98490fb57c 🙂